packages/react-components/src/components/chart/chartOptions/seriesAndYAxis/convertSeriesAndYAxis.ts

```diff
@@ -1,14 +1,14 @@
 import { DataPoint, DataStream, Threshold } from '@iot-app-kit/core';
 import { BarSeriesOption, SeriesOption, YAXisComponentOption } from 'echarts';
 
-import maxBy from 'lodash.maxby';
-import minBy from 'lodash.minby';
+// import maxBy from 'lodash.maxby';
+// import minBy from 'lodash.minby';
 
 import {
   ChartAxisOptions,
   ChartStyleSettingsOptions,
   Visualization,
-  YAxisLegendOption,
+  // YAxisLegendOption,
```

Comment:
Can these comments be removed?

Reply:
Ahh, I thought I cleaned all of these up!